captum/optim/_param/image/transforms.py

```diff
@@ -93,6 +111,16 @@ def __init__(self, transform: Union[str, torch.Tensor] = "klt") -> None:
             )
 
     def forward(self, x: torch.Tensor, inverse: bool = False) -> torch.Tensor:
+        """
+        Args:
+            x (torch.tensor):  A CHW or NCHW RGB or RGBA image tensor.
+            inverse (bool, optional):  Whether to recorrelate or decorrelate colors.
+                Default is set to False.
+        Returns:
+            chw (torch.tensor):  A tensor with it's colors recorrelated or
+                decorrelated.
+        """
+
```

Comment:
The `__init__` for CenterCrop + RandomScale (public methods), RandomSpatialJitter (public methods) need docs for args as well.